package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@silvermine/videojs-chromecast",
-  "version": "1.3.2",
+  "version": "1.3.3",
```

Comment:
I'm not sure why you bumped the version of this plugin. Two things:

 1. if / when it's bumped, it will be _at least_ a minor bump, not just a patch. It might even require a major bump depending on whether this change is backwards-compatible with older versions of VideoJS
 2. version bumps are not included in a feature merge - they're a separate commit

Reply:
https://github.com/silvermine/videojs-chromecast/pull/112/commits/cfc31faf24d16b97bc0c3df4cd538e143643ee75